src/app/zones/views/ZoneCreateView.vue

```diff
@@ -234,6 +270,10 @@
                     <h2 class="form-section-title">
                       {{ t('zones.form.section.scanner.title') }}
                     </h2>
```

Comment:
I know its not part of this PR but I noticed that when the zone has been found, it looks really strange that it still says "Waiting for Zone to be connected...", should we just remove this header?

Reply:
It’s better than not arbitrary having a form section without a header. It’s also unambiguous that the Zone is connected when the widget says so so I don’t see this as a conflict.

Reply:
It doesn’t look broken to me, no.

It doesn’t make sense to change the wording of a section heading conditionally to some state inside the section.

I’m happy to change the title to something not related to the state, but I’m not removing it because that’s inconsistent with the form layout.

Reply:
Ok maybe lets bring it up with @johnharris85 ? Feel free to resolve for now, we can add an issue for this if folks want to tweak.